go.mod

```diff
@@ -1,53 +1,44 @@
 module github.com/openshift/cluster-cloud-controller-manager-operator
 
-go 1.18
+go 1.19
 
 require (
 	github.com/asaskevich/govalidator v0.0.0-20210307081110-f21760c49a8d
 	github.com/go-logr/logr v1.2.3
 	github.com/golangci/golangci-lint v1.49.0
-	github.com/onsi/ginkgo/v2 v2.6.0
-	github.com/onsi/gomega v1.24.0
-	github.com/openshift/api v0.0.0-20230111143458-54592eea5539
-	github.com/openshift/library-go v0.0.0-20230112164258-24668b1349e6
-	github.com/spf13/cobra v1.5.0
+	github.com/onsi/ginkgo/v2 v2.7.0
+	github.com/onsi/gomega v1.24.2
```

Comment:
gomega 1.25 was already released

Reply:
Do you prefer me to update it in this PR or have it in the PR for k8s packages update?

Reply:
Here will be ok i think